otx/algorithms/classification/adapters/openvino/model_wrappers/openvino_models.py

```diff
@@ -86,18 +84,18 @@ def _get_outputs(self):
 
     def postprocess(self, outputs: Dict[str, np.ndarray], meta: Dict[str, Any]):  # pylint: disable=unused-argument
         """Post-process."""
-        logits = outputs[self.out_layer_name].squeeze()
+        logits = outputs[self.out_layer_names[-1]].squeeze()
```

Comment:
Did you check backward compatibility for classification ov inference? 
For example 
1. Export in otx1.3
2. Inference exported openvino.xml in this branch

Reply:
I checked that for classification, but it was rather for debugging purposed.

This PR doesn't bring any changes to openvino.xml (excepting ones that new MO could introduce). All changes in OV wrappers are related to upgrade of ModelAPI version, so old IRs are handled as well (since input/output specs like names and shapes were not changed).